torba/torba/server/db.py

```diff
@@ -422,8 +422,8 @@ def fs_tx_hash(self, tx_num):
     async def fs_block_hashes(self, height, count):
         headers_concat, headers_count = await self.read_headers(height, count)
         if headers_count != count:
-            raise self.DBError('only got {:,d} headers starting at {:,d}, not '
-                               '{:,d}'.format(headers_count, height, count))
+            raise self.DBError(f'only got {headers_count:,d} headers starting '
+                               f'at {height:,d}, not {count:,d}')
```

Comment:
Can you change this to:
```
            raise self.DBError(
                f'only got {headers_count:,d} headers starting at {height:,d}, not {count:,d}')
            )
```

Reply:
@eukreign We could do that, but then that line would become too long (110 characters). That looked too much.

I wasn't sure how long a line was acceptable for this project. What do you think? 110 is fine?

Reply:
yes:

https://github.com/lbryio/lbry-sdk/blob/9bb0ca2f0d7ef428d17ee8bdec89d0bb17f99dca/torba/setup.cfg#L16

Reply:
@eukreign Okay, changing this.